File: Famoser.MassPass.Data/Entities/Communications/Response/Authorization/AuthorizedDevicesResponse.cs
using System.Collections.Generic;
using Famoser.MassPass.Data.Entities.Communications.Response.Base;
using Famoser.MassPass.Data.Entities.Communications.Response.Entitites;

namespace Famoser.MassPass.Data.Entities.Communications.Response.Authorization
{
    public class AuthorizedDevicesResponse : ApiResponse
    {
        public AuthorizedDevicesResponse()
        {
            AuthorizedDeviceEntities = new List<AuthorizedDeviceEntity>();
        }

        public List<AuthorizedDeviceEntity> AuthorizedDeviceEntities { get; set; }
    }
}

File: Famoser.MassPass.Api/src/Models/Response/Authorization/AuthorizedDevicesResponse.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 23.05.2016
 * Time: 09:42
 */

namespace Famoser\MassPass\Models\Response\Authorization;


use Famoser\MassPass\Models\Response\Base\ApiResponse;
use Famoser\MassPass\Models\Response\Entities\AuthorizedDeviceEntity;

class AuthorizedDevicesResponse extends ApiResponse
{
    /**
     * @var \Famoser\MassPass\Models\Response\Entities\AuthorizedDeviceEntity
     */
    public $AuthorizedDeviceEntities;
}